packages/firestore/src/local/shared_client_state.ts

```diff
@@ -532,6 +540,10 @@ export class WebStorageSharedClientState implements SharedClientState {
 
     this.onlineStateKey = createWebStorageOnlineStateKey(this.persistenceKey);
 
+    this.bundleChangedRemoteDocumentsKey = createRemoteDocumentsLoadFromBundleKey(
```

Comment:
`createRemoteDocumentsLoadFromBundleKey` seems like it needs a name update now as well. `createBundleLoadedKey`?

For sake of brevity, I would also change bundleChangedRemoteDocumentsKey to bundleLoadedKey. I seems shorter and more specific.

Reply:
Done.